bin/upgrading/en.dat

```diff
@@ -835,6 +835,8 @@ error.usernameinvalid=This account name contains invalid characters.
 
 error.usernamelong=This account name is too long.  Account names can't be longer than 25 characters.
 
```

Comment:
If you're going to replace a translation string and add the old version to deadphrases.dat, it also needs to be removed from en.dat.